src/generator/__snapshots__/generator.unit.js.snap

```diff
@@ -506,7 +506,7 @@ exports[`generates a CVE playbook if CSAW rule is not found 1`] = `
   become: true
   tasks:
     - name: check for update
-      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q --cve {{ insights_issues | regex_search('(CVE-[0-9]{4}-[0-9]+( --cve CVE-[0-9]{4}-[0-9]+)*)') }}\\"
+      shell: \\"{{ ansible_facts['pkg_mgr'] }} check-update -q {{ insights_issues | regex_search('(--cve (CVE-[0-9]{4}-[0-9]))+') }}\\"
```

Comment:
@semtexzv It looks like the Regex for the CVE templates were changed.  I believe a "+" needs to be added after the [0-9] seeing as currently this is just checking for "CVE-1234-3"